example/tests/test_views.py

```diff
@@ -101,6 +101,7 @@ def test_patch_to_one_relationship(self):
         }
         response = self.client.patch(url, data=json.dumps(request_data), content_type='application/vnd.api+json')
         assert response.status_code == 200, response.content.decode()
+        assert response.data == request_data['data']
```

Comment:
@scottfisk The line fails on `test_patch_to_one_relationship` without the fix. It passes before and after on `test_patch_to_many_relationship`, but I added it just to make sure that a future change doesn't cause it to stop working since only checking a status code of 200 doesn't make any guarantees about the integrity of the data in the response.